Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -351,6 +350,12 @@
           min: 1
         reagent: Histamine
         amount: 4
+      - !type:GenericStatusEffect
+        conditions:
+        - !type:TotalDamage
+          min: 90
+        key: PressureImmunity
```

Comment:
Absolutely not. No pressure immunity, not even with a health threshold. Unless this is removed, this PR will not be merged.

Reply:
Are you also against the space immunity if the threshold is the person being crit? This way people can't use it on themselves, but other people can keep someone stable for a bit longer on the way out of a spaced area.